### PR TITLE
Use dedicated style resource modules

### PR DESCRIPTION
**Pull request by malberts**
Follow-up to https://github.com/ProfessionalWiki/PageApprovals/pull/67

Using the style loading in https://github.com/ProfessionalWiki/PageApprovals/issues/45#issuecomment-2198029240

https://github.com/ProfessionalWiki/PageApprovals/assets/1428594/14cb43a3-5039-4f09-8704-6448404e8dd4

While the current approach does work, it's not the usual way styles are loaded in MediaWiki.

### CODE REVIEW

File: resources/ManageApprovers.css
Comment (malberts) [inline resources/ManageApprovers.css:1]: Nothing was changed here. Just moved out of the template.

File: resources/PageApprovals.css
Comment (malberts) [inline resources/PageApprovals.css:1]: This was moved out of the template and combined with the existing selectors.

File: resources/ManageApprovers.css
Comment (malberts) [inline resources/ManageApprovers.css:1]: Unrelated to this PR, but some of the generic selectors in here could be a bit dangerous. Other/Custom skins might have extra UI elements that can match those selectors.

File: resources/PageApprovals.css [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+	position: absolute;
+	top: 100%;
+	right: 0;
+	background-color: white;
+	box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+	border-radius: 4px;
+	overflow: hidden;
+}
+
+.dropdown-item {
```
Comment (malberts) [inline resources/PageApprovals.css:144]: Unrelated to this PR, but we need to be careful with custom classes that match Bootstrap classes. The Chameleon skin uses Bootstrap and could have unexpected side-effects if we redeclare Bootstrap classes.

Comment (JeroenDeDauw) [reply resources/PageApprovals.css:144]: @waximabbax

**Review by JeroenDeDauw — APPROVED** [review]

File: resources/PageApprovals.css (hunk 1 above)
Comment (waximabbax) [reply resources/PageApprovals.css:144]: I am aware of this but I didn't know `.dropdown-item` was a bootstrap class.

**Pull request merged**